.github/workflows/validate-models.yml

```diff
@@ -9,26 +9,33 @@ jobs:
   validate-models:
     runs-on: ubuntu-latest
     steps:
```

Comment:
I don't think we need logging enabled in the pipeline. The exceptions should explain what went wrong. This is how I have it setup in my fork

https://github.com/digimaun/iot-plugandplay-models/blob/main/.github/workflows/validate-models.yml#L33

Reply:
I'd expect to have the correct LogLevel by default.